client/state/directly/reducer.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	DIRECTLY_INITIALIZE
+} from 'state/action-types';
+import { createReducer } from 'state/utils';
+
+export const isInitialized = createReducer( false, {
+	[ DIRECTLY_INITIALIZE ]: () => true,
+} );
```

Comment:
Yeah it's a weakness of the first implementation that we persist by default if we use a vanilla reducer.

Whatever helpers we use, opt-in only for persistence is _much_ more dev friendly. Folks might not know about persistence, and we avoid future data shape/bad app state bugs.